### DISCUSSION TITLE
Поздравляю, вышло рабочее обновление, но тоже с сюрпризом.

### DISCUSSION THREAD

**Discussion by NikolaichIg** [FarbstoffRSL/RSL-Helper#434] [General]
Печально, но сегодняшнее обновление окончательно добило хелпер. Теперь он даже не запускается.

**Comment by NikolaichIg** [comment]
А нет, был неправ. Теперь он оказывается запускается только при запущенном RSL.
Работает и даже появились 2 загадочных кнопки "Предыдущая волна" и "Следующая волна". Назначение пока непонятно.
Может кто подскажет, как работает в настройках галочка "Activale auto mino scroll"?

**Reply by Dogadai88** [reply]
Вот оно что, а я снёс его из за того что н не запускался)

**Comment by NikolaichIg** [comment]
Разобрался я с этими кнопочками. Это оказывается чертовски полезная штука. Теперь есть возможность узнать все параметры волн в данжах, кроме кланового боса. Давно я хотел узнать характеристики боссов на разных уровнях, теперь это стало доступно.
